src/test/java/jdbc/JdbcTemplateTest.java

```diff
@@ -0,0 +1,85 @@
+package jdbc;
+
+import database.DatabaseServer;
+import database.H2;
+import exception.EmptyResultException;
+import exception.IncorrectResultSizeException;
+import org.junit.jupiter.api.*;
+import persistence.sql.ddl.EntityMetaData;
+import persistence.sql.ddl.Person;
+import persistence.sql.ddl.query.DdlQueryBuilder;
+import persistence.sql.dml.InsertQueryBuilder;
+import test.PersonGenerator;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JdbcTemplateTest {
+
+    Person person = PersonGenerator.getDefualtPerson();
+    JdbcTemplate jdbcTemplate;
+    InsertQueryBuilder insertQueryBuilder;
+    EntityMetaData entityMetaData;
+
+    @BeforeEach
+    void setUp() throws SQLException {
+        entityMetaData = new EntityMetaData(person);
+
+        final DatabaseServer server = new H2();
+        server.start();
+
+        jdbcTemplate = new JdbcTemplate(server.getConnection());
+
+        //테이블 생성
+        DdlQueryBuilder ddlQueryBuilder = DdlQueryBuilder.getInstance();
+        jdbcTemplate.execute(ddlQueryBuilder.createTable(entityMetaData));
+
+        insertQueryBuilder = new InsertQueryBuilder();
+    }
+
+
+    @Test
+    @DisplayName("result data 가 없을 때 테스트")
+    void noResultTest() {
+        String insertQuery = insertQueryBuilder.create(entityMetaData);
+//        jdbcTemplate.execute(insertQuery);
+
+        assertThrows(EmptyResultException.class, this::queryForObject);
+    }
+
+
+    @Test
+    @DisplayName("result data 가 1개일때 테스트")
+    void singleResultTest() {
+        String insertQuery = insertQueryBuilder.create(entityMetaData);
+        jdbcTemplate.execute(insertQuery);
+        queryForObject();
+    }
+
+    @Test
+    @DisplayName("result data 가 2개 이상일 때 테스트")
+    void moreThenTwoResult() {
```

Comment:
테스트에서는 어떤 상황일 때, 어떻게 되는 지 자세하게 적어주시면 동료 개발자들이 쉽게 알아볼 수 있습니다 :)
```suggestion
    @Test
    @DisplayName("조회된 데이터가 2개 이상이면 예외를 발생한다")
    void moreThenTwoResult() {
```